seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -267,12 +267,12 @@
       "tests": [
         "// Test user did replace var keyword",
         "// Test squaredIntegers is const",
-        "// Test squaredIntegers is an array",
-        "// Test squaredIntegers is [16, 1764, 36]",
-        "// Test function keyword was not used",
-        "// Test arrow => was used",
-        "// Test loop was not used",
-        "// Test map and filter were used"
+        "assert(Array.isArray(squaredIntegers), 'message: squaredIntegers should be an array');",
+        "assert(squaredIntegers[0] === 16 && squaredIntegers[1] === 1764 && squaredIntegers[2] === 36, 'message: squaredIntegers should be [16, 1764, 36]');",
+        "assert(!code.match(/function/g), 'message: function keyword should not be used');",
+        "assert(code.match(/=>/g), 'message: arrow => should be used');",
+        "assert(!code.match(/(for)|(while)/g), 'message: loop should not be used');",
+        "assert(code.match(/map/g) && code.match(/filter/g), 'message: map and filter should be used');"
       ],
```

Comment:
@MiloATH Also, when adding tests, we should try our best to keep everything formatted correctly, so that we will have to go over these less times to get them in good shape. In this case, what's missing is `<code>` tags surrounding keywords  and code snippets in your test case messages, i.e. `map`, `filter`, `=>` (even though I think that test should probably remain commented out for now), `function`, `squaredIntegers`, `[16, 1764, 36]`, etc. 

This isn't the biggest deal in the world, but it helps keep things clean and consistent across the site, and ultimately makes for a better user experience. 

So final assessment, if you could:
- comment out both tests again involving the `function` keyword and `=>` syntax until we have working support for ES6 tests
- fix formatting issues

Once this is done, I think we'd be fine to merge this in since it does not involve any ES6 tests, not a bad idea to fill in the blanks where we can until ES6 support is added. Thanks for your help with this!